src/servicesHost.ts

```diff
@@ -27,8 +27,17 @@ function makeServicesHost(
     const resolveSync = makeResolver(loader.options);
 
     const moduleResolutionHost = {
-        fileExists: (fileName: string) => utils.readFile(fileName) !== undefined,
-        readFile: (fileName: string) => utils.readFile(fileName),
+        fileExists: (fileName: string) => {
```

Comment:
Quite the contrary :-) I'll get to these changes within the next couple of days (I am actually at my day job right now).